src/compas_timber/utils/__init__.py

```diff
@@ -305,7 +305,10 @@ def is_polyline_clockwise(polyline, normal_vector):
         True if the polyline is clockwise, False otherwise.
 
     """
-    assert polyline[0] == polyline[-1], "Polyline should be closed"
+    # make sure the polyline is closed
+    if not polyline[0] == polyline[-1]:
+        polyline.append(polyline[0])
```

Comment:
As a client, I wouldn't expect a function called `is_polyline_clockwise` to modify what I give it.
if polyline has to be closed for the check, but the function should support both closed and opened polylines then make the check on a copy.